openmdao/core/system.py

```diff
@@ -4215,7 +4215,12 @@ def _get_input_from_src(self, name, abs_names, conns, units=None, indices=None,
                 try:
```

Comment:
Can you add a docstring to `_get_input_from_src` ?

Reply:
done